16.0 t 32664 jjs

jjscarafia opened the pull request.

jjscarafia committed:
Revert "[REM] account_invoice_move_currency: deprecated"

This reverts commit 265f4c4d50bbb4e53a2d7378c705645e977dee07.

jjscarafia committed:
[MIG] account_invoice_move_currency: Migration to 16.0

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-invoicing/pull/156).

jjscarafia committed:
[REF] _move_currency: simplified
Depreciamos todo lo que impactaba en el apunte contable porque era la parte mas heavy y dejamos
solo a modo informativo una moneda secundaria y su cotizacion
